static/app/views/issueList/groupSearchViewTabs/draggableTabBar.tsx

```diff
@@ -301,10 +301,30 @@ export function DraggableTabBar({
     }
   };
 
-  const handleNewViewSaved: NewTabContext['onNewViewSaved'] = useCallback(
-    () => (label: string, query: string, saveQueryToView: boolean) => {
+  const handleNewViewsSaved: NewTabContext['onNewViewsSaved'] = useCallback<
```

Comment:
Rather than creating a separate function, I refactored the existing context function (which takes the suggestions and turns them into views) to accept multiple suggestions and bulk creates the views for them. By default, the first tab in the suggestion list will be selected when they're all added, but this can be easily changed.